packages/core/package.json

```diff
@@ -41,21 +36,21 @@
   },
   "devDependencies": {
     "@arethetypeswrong/cli": "^0.15.3",
-    "@shikijs/transformers": "^1.9.1",
+    "@shikijs/transformers": "^3.0.0",
     "@types/node": "^20.14.9",
     "hast-util-to-html": "^9.0.1",
     "mdast-util-to-hast": "^13.2.0",
     "prettier": "^3.3.2",
     "remark": "^15.0.1",
-    "shiki": "^1.9.1",
+    "shiki": "^3.0.0",
     "tsup": "^8.1.0",
     "tsx": "^4.15.7",
     "typescript": "^5.5.2",
     "vite": "^5.3.1",
     "vitest": "^1.6.0"
   },
   "peerDependencies": {
-    "shiki": "^1.3.0"
+    "shiki": "^1.0.0 || ^2.0.0 || ^3.0.0"
```

Comment:
This should be `^3.0.0` because `getHighlighter` was renamed to `getSingletonHighlighter`. This means v1 and v2 can't be supported anymore.